src/main/java/me/botsko/prism/serializers/items/ItemStackSerializer.java

```diff
@@ -0,0 +1,295 @@
+package me.botsko.prism.serializers.items;
+
+import me.botsko.prism.Prism;
+import me.botsko.prism.actions.GenericAction;
+import me.botsko.prism.utils.EntityUtils;
+import me.botsko.prism.utils.ItemUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.Color;
+import org.bukkit.DyeColor;
+import org.bukkit.FireworkEffect;
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.Tag;
+import org.bukkit.block.BlockState;
+import org.bukkit.block.Container;
+import org.bukkit.block.banner.Pattern;
+import org.bukkit.block.banner.PatternType;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.BannerMeta;
+import org.bukkit.inventory.meta.BlockStateMeta;
+import org.bukkit.inventory.meta.BookMeta;
+import org.bukkit.inventory.meta.EnchantmentStorageMeta;
+import org.bukkit.inventory.meta.FireworkEffectMeta;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.inventory.meta.LeatherArmorMeta;
+import org.bukkit.inventory.meta.SkullMeta;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Created by Narimm on 29/10/2020.
+ */
+public class ItemStackSerializer {
+    public int amt;
+    public String name;
+    public int color;
+    public String owner;
+    public String[] enchs;
+    public String by;
+    public String title;
+    public String[] lore;
+    public String[] content;
+    public String slot = "-1";
+    public int[] effectColors;
+    public int[] fadeColors;
+    public boolean hasFlicker;
+    public boolean hasTrail;
+    public short durability = 0;
+    public Map<String, String> bannerMeta;
+    public String material;
+
+    public static ItemStackSerializer createItemStackSerialized(ItemStack item) {
```

Comment:
Codacy found an issue: [The method 'createItemStackSerialized(ItemStack)' has an NPath complexity of 32256, current threshold is 200](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6368169)